osf/utils/workflows.py

```diff
@@ -423,7 +423,7 @@ def db_name(self):
         'trigger': 'resubmit',
         'source': [CollectionSubmissionStates.REJECTED, CollectionSubmissionStates.REMOVED],
         'dest': CollectionSubmissionStates.ACCEPTED,
-        'before': ['_validate_resubmit'],
```

Comment:
If you're a contributor on an unmoderated collection you don't need to be an admin.